js/constants/constants.js

```diff
@@ -88,6 +88,21 @@ module.exports = {
     SHADOW: "Shadow"
   },
 
+  TOOLTIPS: {
```

Comment:
@FernandoCuevas Looks like that you need to use Utils.getLocalizedString() method to get a proper localized text.
For example:

var liveText = Utils.getLocalizedString(this.props.language, CONSTANTS.SKIN_TEXT.LIVE, this.props.localizableStrings);